thirdparty_containers/payara-mp-tck/dockerfile/payara-mp-tck.sh

```diff
@@ -33,26 +33,10 @@ else
 fi
 
 java -version
-cd ${PAYARA_HOME}/
 
-#Start Payara server
-${PAYARA_HOME}/payara5/bin/asadmin start-domain
+TEST_OPTIONS=$1
 
-#Start MicroProfile-Metrics TCK
-cd ${PAYARA_HOME}/MicroProfile-TCK-Runners/MicroProfile-Metrics/tck-runner
-mvn -Ppayara-server-remote test
+cd ${PAYARA_HOME}/MicroProfile-TCK-Runners
+mvn clean verify -Dpayara.version=5.184 $TEST_OPTIONS
```

Comment:
I think eventually we will want to pass in APP_VERSION as a param (as 5.184 will go out of date at some point), but can be a follow-up PR.